x-pack/plugin/core/src/main/resources/logs-template.json

```
@@ -4,6 +4,7 @@
   "data_stream": {},
   "composed_of": [
     "logs-mappings",
+    "data-streams-mappings",
     "logs-settings"
```

I think we should apply the `data-streams-mappings` settings first, so that any changes made to the `logs-mappings` component template always take precedence over the generic data stream mappings.

```suggestion
    "data-streams-mappings",
    "logs-mappings",
    "logs-settings"
```

What do you think?

---

Actually this would break tests because then the dynamic template that maps strings as keywords would take precedence over the dynamic template that maps message fields as `match_only_text`. In order to change the order, we would also need to configure `unmatch:message` on the default dynamic template that maps strings as keywords. What is your preference?